dune-project

```diff
@@ -15,30 +15,4 @@
 
 (source (github ocaml/ocaml-lsp))
 
-(generate_opam_files true)
```

Comment:
Had to remove generation of opam files for now to be able to provide custom build commands.

Reply:
There's no need to remove it. It's possible to override the build step only in the generated .opam file. See how we do it in dune for example https://github.com/ocaml/dune/blob/master/dune.opam.template

Reply:
Whoa, didn't know dune supports it. Fixed.